README.md

```diff
@@ -2,4 +2,8 @@ HIFIS Marketplace
 ===================
 ![](https://github.com/hifis-cloud/hifis-marketplace/workflows/CI/badge.svg)
 
-An entry point web application for scientific related services from various vendors.
\ No newline at end of file
+An entry point web application for scientific related services from various vendors.
```

Comment:
This change in this line is unrelated to this patch.

Reply:
ditto

Reply:
Looking in the code it seems that the change in this line is only the newly added CR/LF, or do I misunderstand your comment?

Reply:
I see.